package.json

```diff
@@ -11,7 +11,9 @@
     "build": "webpack --config webpack.prod.js",
     "start": "webpack-dev-server --hot --color --progress --info=true --config webpack.dev.js",
     "test": "jest",
-    "lint": "tslint -c ./tslint.json --project .",
+    "tslint": "tslint -c ./tslint.json --project .",
+    "eslint": "eslint --ext tsx ./src/",
```

Comment:
I wonder if we should include the `.js` file extension here too, in case people are bringing in JS code that they haven't converted to TS yet or something... I'm trying to determine if there's a legitimate reason to have JS files around if you've started your project with the TS seed project. Maybe not.

Reply:
Absolutely agree, good catch! TypeScript is still new and we should ensure our tooling supports both JS and TS both as first-class citizens.